src/commons/assessment/Assessment.tsx

```diff
@@ -60,6 +61,23 @@ const Assessment: React.FC = () => {
   const { courseId, role, assessmentOverviews: assessmentOverviewsUnfiltered } = useSession();
   const dispatch = useDispatch();
 
+  useEffect(() => {
+    if (assessmentOverviewsUnfiltered && courseId) {
+      sendToWebview(
```

Comment:
Is this why we are sending everything (including upcoming ones)? Looks good for now but can always make this minor change in the future.